.gitignore

```diff
@@ -63,24 +63,7 @@ obj/
 /out/
 
 # User-specific configurations
-.idea/libraries/
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/.name
-.idea/compiler.xml
-.idea/copyright/profiles_settings.xml
-.idea/encodings.xml
-.idea/misc.xml
-.idea/modules.xml
-.idea/scopes/scope_settings.xml
-.idea/dictionaries
-.idea/vcs.xml
-.idea/jsLibraryMappings.xml
-.idea/datasources.xml
-.idea/dataSources.ids
-.idea/sqlDataSources.xml
-.idea/dynamic.xml
-.idea/uiDesigner.xml
+.idea/
```

Comment:
Well, the `.idea/` really just contains configs (project settings) for your IntelliJ/Android Studio. If a startup/company has a company-wide default project setting, you would want to keep that. However, we don't. I think we should remove it. 

I also just played around with gitignore.io and I realized that they give different .gitignore files for Android and Android Studio (understandably, so. The latter does not ignore `.idea/`). I usually use both tags so they ignore pretty much every file in the `.idea/`. I think you should just go ahead and gitignore `/.idea/*`

cc: @prabirmsp @as-f